### PR TITLE
Docs for Delivery API V2 incl. new property expansion and limiting

### PR DESCRIPTION
**Pull request by kjac**
## Description

Added docs for the new property expansion and limiting feature in the Delivery API + updated all relevant V13 Delivery API docs to reference V2 of the Content and Media Delivery APIs (which are introduced to handle the new expansion and limiting feature).

### Summary.md?

I'm not entirely sure I made the correct updates to `13/umbraco-cms/SUMMARY.md` ... I just added all the Delivery API articles there, including the newly introduced one (Property expansion and limiting). Please reach out if something needs fixing.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

V13 RC2 (hopefully)

## Deadline (if relevant)

15/11 if possible... provided the feature makes the cut for V13 RC2. Please don't merge until we have explicitly given the OK 😄

### CODE REVIEW

**Review by elit0451 — APPROVED** [review]
Great docs for a great feature 🥳 🙌

File: 13/umbraco-cms/SUMMARY.md [hunk 1]
```diff
@@ -20,6 +20,14 @@
   * [External login providers](reference/security/external-login-providers.md)
   * [Two-factor Authentication](reference/security/two-factor-authentication.md)
 
+* [Content Delivery API](reference/content-delivery-api/README.md)
+    * [Custom property editors support](reference/content-delivery-api/custom-property-editors-support.md)
+    * [Extension API for querying](reference/content-delivery-api/extension-api-for-querying.md)
+    * [Media Delivery API](reference/content-delivery-api/media-delivery-api.md)
+    * [Protected content in the Delivery API](reference/content-delivery-api/protected-content-in-the-delivery-api.md)
+    * [Additional preview environments support](reference/content-delivery-api/additional-preview-environments-support.md)
```
Comment (jonat123) [inline 13/umbraco-cms/SUMMARY.md:28]:
```suggestion

```

File: 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md [hunk 2]
```diff
@@ -0,0 +1,1376 @@
...
+{% hint style="info" %}
+When working with property expansion and limiting in API queries, there are two rules of thumb to keep in mind:
+
+1. Expandable properties are _not_ expanded by default. They must be expanded explicitly.
+2. All properties are included in the API output by default. We can apply limiting to limit the included properties.
+{% endhint %}
+
+## Working with picker editors
+
+In the following, we will be querying a content tree with blog posts and blog authors:
```
Comment (jonat123) [inline 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md:32]:
```suggestion
In the following example, we will be querying a content tree with blog posts and blog authors:
```

Comment (jonat123) [reply 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md:32]: @kjac Seemed like something was missing here, so i added this suggestion :-)

File: 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md [hunk 3]
```diff
@@ -0,0 +1,1376 @@
...
+
+## Working with block-based editors
+
+{% hint style="info" %}
+If you are not familiar with block-based editors, please refer to [this article](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor) for the general concepts of these.
+{% endhint %}
+
+In the API output, a block has little value without its contained properties. Therefore, the content and settings properties of blocks are always included in the output. However, these properties are not expanded. As such, we can apply expansion and limiting to the contained properties.
+
+In the following we'll request different types of articles, all of which are located under a root content item called "Articles":
```
Comment (jonat123) [inline 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md:545]:
```suggestion
In the following example we'll request different types of articles, all of which are located under a root content item called "Articles":
```

File: 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md (hunk 2 above)
Comment (kjac) [reply 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md:32]: We should do plural then... the querying goes on for several examples. I'll amend 😄

File: 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md (hunk 3 above)
Comment (kjac) [reply 13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md:545]: Same as above 😆

File: 13/umbraco-cms/SUMMARY.md (hunk 1 above)
Comment (jonat123) [reply 13/umbraco-cms/SUMMARY.md:28]: @kjac This looks like the only article in this section with no changes, so I have suggested to remove it from the summary 😄

**Review by jonat123 — CHANGES_REQUESTED** [review]
@kjac Looks really great!

I've made a few suggestions, please let me know what you think :D

File: 13/umbraco-cms/SUMMARY.md (hunk 1 above)
Comment (kjac) [reply 13/umbraco-cms/SUMMARY.md:28]: @jonat123 are you sure about this? The article is in v13:

image

Comment (kjac) [reply 13/umbraco-cms/SUMMARY.md:28]: Ah so it has to do with which ones that have changed? Thanks 😛 I have no idea how these summaries work across versions.

**Comment by kjac** [comment]
@jonat123 this should be ready to go. RC2 is out now 😄

**Review by jonat123 — APPROVED** [review]

**Pull request merged**